charts/common/templates/helpers/_labels.tpl

```diff
@@ -0,0 +1,63 @@
+{{- /*
+library.labelize takes a dict or map and generates labels.
+Values will be quoted. Keys will not.
+Example output:
+  first: "Matt"
+  last: "Butcher"
+*/ -}}
+{{- define "library.labelize" -}}
+{{- range $k, $v := . }}
+{{ $k }}: {{ $v | quote }}
+{{- end -}}
+{{- end -}}
+
+{{- /*
+library.labels.standard prints the standard Helm labels.
+The standard labels are frequently used in metadata.
+
+Based on https://docs.helm.sh/chart_best_practices/#standard-labels
+*/ -}}
+{{- define "library.labels.standard" -}}
+app.kubernetes.io/name: {{ template "library.name" . }}
+helm.sh/chart: {{ template "library.chartrefshort" . }}
+helm.nsm.bedag.ch/chartname: {{ .Chart.Name | quote }}
```

Comment:
I would change all labels with `nsm.bedag.ch` to something different (we can discuss this)